src/exoticatechnologies/modifications/exotics/impl/HackedMissileForge.kt

```diff
@@ -19,13 +19,14 @@ import java.awt.Color
 
 class HackedMissileForge(key: String, settings: JSONObject) : Exotic(key, settings) {
     override var color = Color(0xFF8902)
+
+    override fun getBasePrice() = COST_CREDITS.toInt()
+
     override fun canAfford(fleet: CampaignFleetAPI, market: MarketAPI?): Boolean {
-        return (Utilities.hasItem(fleet.cargo, ITEM)
-                && fleet.cargo.credits.get() >= COST_CREDITS)
+        return Utilities.hasItem(fleet.cargo, ITEM)
```

Comment:
can swear i actually bought one for 250k but maybe i didn't have the latest deployment so i'll fix this issue the next time i run into it, if i run into it.

Reply:
for what it's worth, HackedMissileForge costs 150k